src/api/response/iiif/manifest.js

```diff
@@ -259,6 +256,37 @@ function transform(response) {
       }
     }
 
+    /** Add logo manually (w/o IIIF Builder) */
+    const nulLogo = {
+      id: "https://iiif.dc.library.northwestern.edu/iiif/2/00000000-0000-0000-0000-000000000003/full/pct:50/0/default.webp",
+      type: "Image",
+      format: "image/webp",
+      height: 139,
+      width: 1190,
+    };
+    jsonManifest.logo = [nulLogo];
+
+    /** Add provider manually (w/o IIIF Builder) */
+    const provider = {
+      id: "https://dc.library.northwestern.edu/",
```

Comment:
This should be the providing organization/institution so a bit higher level than just DC. Maybe @davidschober could chime in. My vote would be for LOC authority id if we have one. I think this plays better with JSON-LD. So, this could be:

- Library of Congress authority ID, not sure what ours is? ex: https://iiif.io/api/cookbook/recipe/0234-provider/
- Homepage of Libraries https://www.library.northwestern.edu
- Or even the about page of the Libraries https://www.library.northwestern.edu/about/

```suggestion
      id: "https://www.library.northwestern.edu/",
```